docs/modules/adapterRemoval.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
Please add a reference to this file in `docs/README.md` (required to be built into the website docs).